common/random.h

```diff
@@ -39,8 +39,8 @@ namespace Common {
 class String;
 
 /**
- * Simple random number generator. Although it is definitely not suitable for
- * cryptographic purposes, it serves our purposes just fine.
+ * Xorshift PRNG. Replaces the previous PRNG, as
+ * the cycles within it were detrimental to gameplay.
```

Comment:
```suggestion
 * Xorshift random number generator. Although it is definitely not suitable for
 * cryptographic purposes, it serves our purposes just fine.
```
I don't think we need information about the previous state of the code in our doxygen documentation. What matters is the current state. Also we might want to avoid acronyms as some developers who simply want a random number for their engine might not know what `PRNG` means.

Reply:
Fixed in commit 17c0c6f